src/Raven.Server/Dashboard/MachineResourcesNotificationSender.cs

```diff
@@ -21,13 +19,23 @@ public class MachineResourcesNotificationSender : BackgroundWorkBase
         private readonly TimeSpan _notificationsThrottle;
 
         private DateTime _lastSentNotification = DateTime.MinValue;
+        private static byte[][] _buffers;
+        private readonly SmapsReader _smapsReader;
 
         public MachineResourcesNotificationSender(string resourceName,
             ConcurrentSet<ConnectedWatcher> watchers, TimeSpan notificationsThrottle, CancellationToken shutdown)
             : base(resourceName, shutdown)
         {
             _watchers = watchers;
             _notificationsThrottle = notificationsThrottle;
+
+            if (PlatformDetails.RunningOnLinux)
+            {
+                var buffer1 = ArrayPool<byte>.Shared.Rent(SmapsReader.BufferSize);
+                var buffer2 = ArrayPool<byte>.Shared.Rent(SmapsReader.BufferSize);
+                _buffers = new[] { buffer1, buffer2 };
```

Comment:
This _must_ be in a local varaible or temp field.

Reply:
done